omnisolver/samplers/van.py

```
@@ -0,0 +1,188 @@
+import dimod
+import torch
+import time
+import numpy as np
+from functools import partial
+import random
+from ...van_orig.src_hop_sk.sk import SKModel
+from ...van_orig.src_hop_sk.made2 import MADE
+
+class VanSampler(dimod.Sampler):
+    """Implementatoin of simple random-sampler."""
+
+    def __init__(self, **kwargs):
```

I know this is how it's done in original VAN, but what do you think about handling unused keyword args or perhaps changing this to named args completely?

---

It's a stylistic choice, I don't like having more then 3-5 named args.

---

Ok, let's leave it that way but let's also add a check so that if `kwargs` contain unknown argument it raises an error instead of ignoring it (which is equivalent to checking if `kwargs` are empty at the end of the `__init__`).